src/main/java/uk/gov/justice/probation/courtcaseservice/jpa/repository/HearingRepositoryFacade.java

```diff
@@ -108,15 +112,26 @@ public List<HearingEntity> findByCourtCodeAndHearingDay(String courtCode, LocalD
     public Optional<LocalDateTime> findLastModifiedByHearingDay(String courtCode, LocalDate hearingDay) {
         return hearingRepository.findLastModifiedByHearingDay(courtCode, hearingDay);
     }
+
+    @Transactional
```

Comment:
https://github.com/ministryofjustice/court-case-service/blob/7a9b1d144a0f0e1448c5e9fdfb78fde5bd9705a5/src/main/java/uk/gov/justice/probation/courtcaseservice/jpa/repository/HearingRepositoryFacade.java#L22
How will this interact with the @Transactional on the class?

Reply:
I think I will leave it on to be sure.